src/CMakeLists.txt

```diff
@@ -425,38 +474,52 @@ if(HUB_ENABLE_TESTS AND HUB_ENABLE_COVERAGE)
         else()
             set(parentMacro "${parentName}-")
         endif()
-        if(NOT EXISTS "${PROJECT_SOURCE_DIR}/tests/test-${parentMacro}${name}.cpp")
-            message("parentMacro :${parentMacro}")
-            message("name :${name}")
-            message(FATAL_ERROR "test-${parentMacro}${name}.cpp not found in tests directory")
-        endif()
+        # if(NOT EXISTS "${PROJECT_SOURCE_DIR}/tests/${parentName}/test-${parentMacro}${name}.cpp")
```

Comment:
All source (cpp) file have to be a corresponding test. I'll do this in the future. Issue reported.